compat/test/browser/suspense.test.js

```diff
@@ -772,8 +788,8 @@ describe('suspense', () => {
 				expect(scratch.innerHTML).to.eql(
 					`<div>Hello first 2</div><div>Hello second 2</div>`
 				);
-				expect(Suspender1.prototype.render).to.have.been.calledThrice;
-				expect(Suspender2.prototype.render).to.have.been.calledThrice;
+				expect(Suspender1.prototype.render).to.have.been.calledTwice;
+				expect(Suspender2.prototype.render).to.have.been.calledTwice;
```

Comment:
This behavior change is an example of my previous comment where we no longer rerender intermediate components. Here, Suspender doesn't throw. It renders a lazy child which throws. So Suspender's child is the component that re-renders an additional time here. Will try to fix this so we can assert the right thing here.

Reply:
Rewrote this test to directly spy Lazy which actually throws.